builder/common_builder/common_builder.go

```diff
@@ -104,7 +104,7 @@ func (b *CommonBuilder) Run(hostData *model.HostData) {
 			b.log.Panicf("Can't set current user for fetcher, err: [%v]", err)
 		}
 
-		hostData.Extra.ExadataDevices = b.getExadataDevices()
+		hostData.Extra.Exadata.Devices = b.getExadataDevices()
```

Comment:
Are you sure that Exadata is initialized?

Reply:
Since Exadata is a struct, it is initialized at its zero value, a struct with all its field zeroed, which should be fine!

Reply:
No, because hostData.Extra.Exadata should be a pointer to a Exadata